partner-chains-cli/src/config.rs

```diff
@@ -376,11 +377,18 @@ pub struct SidechainParams {
 	pub governance_authority: MainchainAddressHash,
 }
 
+#[derive(Deserialize, Serialize, Parser, Clone, Debug)]
+pub struct NativeTokenConfig {
+	native_token_policy: String,
+	native_token_asset_name: String,
```

Comment:
cosmetic: it could easily be `AssetConfig` with field `policy_id` and `asset_name`

> An asset is uniquely identified by an asset ID, which is a pair of both the policy ID and asset name.